apps/design-land/src/app/button/button.component.html

```diff
@@ -0,0 +1,37 @@
+<h1>Daff Button</h1>
+
+<h2>Basic Button</h2>
```

Comment:
You're missing the `a` variants of all these buttons.

Reply:
There's no stylistic difference between <button> and <a>.

Reply:
Since `a` tags have different implementations than `button` across browsers (hover effects, focus, etc), I disagree.

Reply:
updated 3493d20